.travis.yml

```diff
@@ -47,4 +47,13 @@ deploy:
     skip_cleanup: false
     script: FIREFOX_WD_URL=http://localhost:4444/wd/hub CHROME_WD_URL=http://localhost:4445/wd/hub npm run lerna:publish
     on:
-      tags: true
\ No newline at end of file
+      tags: true
+  - provider: pages
+    repo: sakuli/apidoc
+    local_dir: ./docs
+    skip_cleanup: true
+    github_token: $GITHUB_TOKEN
+    keep_history: true
+    on:
+      all_branches: true
+      condition: $TRAVIS_BRANCH =~ ^(master|feature/57/apidocs)$
```

Comment:
For the sake of cleanliness, we could remove the feature branch in the deployment specifier.